zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/ThreadLocalSessionsServiceImpl.java

```diff
@@ -0,0 +1,149 @@
+     * @return the {@link Session} from thread local or <code>null</code> if no session is found.
+     *
+     * @deprecated Since the new JWT sessions implementation can only get the session of the current user, a single
+     *             {@link this#get()} method is provided. Once migration to the new JWT sessions is completed all
+     *             references to this method should be updated to use the {@link this#get()} instead.
+     *
+     * TODO: Write out usage of this method prior to JWT migration
+     */
+    @Deprecated
+    @Override
+    public Session get(String id) {
+        return get();
+    }
+
+    /**
+     * Get a {@link Session} session object from thread local.
+     *
+     * @param req the {@link HttpServletRequest} to get the session object from thread local for.
+     * @return the {@link Session} from thread local or <code>null</code> if no session is found.
+     *
+     * @deprecated Since the new JWT sessions implementation can only get the session of the current user, a single
+     *             {@link this#get()} method is provided. Once migration to the new JWT sessions is completed all
+     *             references to this method that are not simply repeating the
+     *             {@link com.github.onsdigital.zebedee.filters.AuthenticationFilter} should be should be updated to
+     *             use {@link this#get()} instead. If the call is duplicating the filter, then it should be removed
+     *             so as not to waste compute and request latency.
+     *
+     * TODO: Write out usage of this method prior to JWT migration
+     */
+    @Deprecated
+    @Override
+    public Session get(HttpServletRequest req) {
+        return get();
+    }
+
+    /**
+     * Get a {@link Session} session object from thread local.
+     *
+     * @return the {@link Session} from thread local or <code>null</code> if no session is found.
+     */
+    @Override
+    public Session get() {
+        UserDataPayload jwtDetails = store.get();
+        if (jwtDetails == null) {
+            return null;
+        }
+
+        return new Session(jwtDetails);
+    }
+
+    /**
+     * Verify the session and store in ThreadLocal store.
+     *
+     * @param token - the access token to be verified and stored.
+     * @throws SessionsException for any problem verifying a token or storing a session in ThreadLocal
+     * @throws IOException if a filesystem error occurs
+     */
+    @Override
+    public void set(String token) throws SessionsException, IOException {
+        // Ensure that any existing session is clear in case this is a recycled thread
+        resetThread();
+
+        if (StringUtils.isBlank(token)) {
+            throw new SessionsException(ACCESS_TOKEN_REQUIRED_ERROR);
+        }
+
+        Session session = super.get(token);
+        if (session == null) {
+            throw new SessionsException(ACCESS_TOKEN_EXPIRED_ERROR);
+        }
+
+        ArrayList<String> teams = new ArrayList<>();
```

Comment:
(minor) This could be declared as a more generic List instead

```suggestion
        List<String> teams = new ArrayList<>();
```